decompiler/sl2decompiler.py

```diff
@@ -390,7 +390,10 @@ def print_keywords_and_children(self, keywords, children, lineno, needs_colon=Fa
         if atl_transform is not None:
             # "at transform:", possibly preceded by other keywords, and followed by an ATL block
             # TODO this doesn't always go at the end. Use line numbers to figure out where it goes
-            if not wrote_colon and lineno is not None:
+
+            # BUG: If there was a "at ..." stmt before a transform, a wrong colon was added.
+            # In tests worked this so far
+            if not wrote_colon and lineno is not None and not has_block:
```

Comment:
That's wrong what i did here:
My fix makes the wrong colon gone, but the real reason is the "at transform" in our current case is written 26 lines to late, even after its children if see this right. This causes the wrong colon on the end of the line before it i guess.

I should probably make a issue for this.